packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -92,7 +92,13 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
         );
 
         this.gcDetailsInInitialSummaryP = new LazyPromise(async () => {
-            return getInitialGCSummaryDetailsFn ? getInitialGCSummaryDetailsFn() : { usedRoutes: [] };
+            // back-compat: 0.32. getInitialGCSummaryDetailsFn() returns undefined in 0.31. Remove undefined check
+            // when N > 34.
+            let gcSummaryDetails: IGarbageCollectionSummaryDetails | undefined;
```

Comment:
You can use optional chaining for functions too 😲 (if you want)
```
const gcSummaryDetails = await getInitialGCSumaryDetailsFn?.();
return gcSummaryDetails ?? { usedRoutes: [] };
```

Reply:
Cool. Good suggestion!